test fail looks real:
```
======================================================================
ERROR: test_incomplete_report_submission (custom.ewsghana.tests.test_input_stock_view.TestInputStockView)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/custom/ewsghana/tests/test_input_stock_view.py", line 268, in test_incomplete_report_submission
    self.assertRedirects(response, url)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/test/testcases.py", line 287, in assertRedirects
    redirect_response = response.client.get(path, QueryDict(query))
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/test/client.py", line 473, in get
    response = super(Client, self).get(path, data=data, **extra)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/test/client.py", line 280, in get
    return self.request(**r)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/test/client.py", line 444, in request
    six.reraise(*exc_info)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/dispatcher.py", line 240, in dispatch
    return self.dispatch_with_priv(request, *args, **kwargs)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/accounting/decorators.py", line 46, in wrapped
    request, *args, **kwargs
  File "submodules/django-prbac-src/django_prbac/decorators.py", line 21, in wrapped
    return fn(request, *args, **kwargs)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/dispatcher.py", line 247, in dispatch_with_priv
    return super(CustomProjectReportDispatcher, self).dispatch(request, *args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/domain/decorators.py", line 213, in __outer__
    return additional_decorator(__inner__)(request, domain, *args, **new_kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/domain/decorators.py", line 80, in _inner
    return view_func(req, domain_name, *args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/domain/decorators.py", line 210, in __inner__
    return func(cls, request, *args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/dispatcher.py", line 223, in dispatch
    return super(ProjectReportDispatcher, self).dispatch(request, *args, **kwargs)
  File "submodules/dimagi-utils-src/dimagi/utils/decorators/datespan.py", line 46, in wrapped_func
    return f(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/dispatcher.py", line 134, in dispatch
    report = cls(request, domain=domain, **report_kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/generic.py", line 133, in __init__
    self._update_initial_context()
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/generic.py", line 444, in _update_initial_context
    report_subtitles=self.report_subtitles,
  File "/home/travis/build/dimagi/commcare-hq/custom/ewsghana/reports/__init__.py", line 181, in report_subtitles
    if self.is_rendered_as_email:
AttributeError: 'StockLevelsReport' object has no attribute 'is_rendered_as_email'
```

--- a/custom/ewsghana/reports/__init__.py
+++ b/custom/ewsghana/reports/__init__.py
@@ -164,7 +164,7 @@ class MultiReport(DatespanMixin, CustomProjectReport, ProjectReportParametersMix
     is_exportable = False
     base_template = 'ewsghana/base_template.html'
     is_rendered_as_email = False
-    
+
     @property
     @memoized
     def active_location(self):

--- a/custom/ewsghana/reports/__init__.py
+++ b/custom/ewsghana/reports/__init__.py
@@ -163,7 +163,8 @@ class MultiReport(DatespanMixin, CustomProjectReport, ProjectReportParametersMix
     printable = True
     is_exportable = False
     base_template = 'ewsghana/base_template.html'
-
+    is_rendered_as_email = False
+    
     @property
     @memoized
     def active_location(self):

--- a/custom/ewsghana/reports/__init__.py
+++ b/custom/ewsghana/reports/__init__.py
@@ -222,7 +222,7 @@ class MultiReport(DatespanMixin, CustomProjectReport, ProjectReportParametersMix
             else:
                 program_id = 'all'
 
-            url = '%s?location_id=%s&filter_by_program=%s&halo=testtest' % (
+            url = '%s?location_id=%s&filter_by_program=%s' % (
                 url,
                 dm.location_id if dm.location_id else '',
                 program_id if program_id else ''

--- a/custom/ewsghana/reports/specific_reports/stock_status_report.py
+++ b/custom/ewsghana/reports/specific_reports/stock_status_report.py
@@ -2,8 +2,7 @@ from datetime import timedelta
 from django.db.models.aggregates import Count
 from corehq.apps.commtrack.models import StockState
 from corehq.apps.locations.models import SQLLocation
-from corehq.apps.products.models import SQLProduct, Product
-from corehq.apps.programs.models import Program
+from corehq.apps.products.models import SQLProduct
 from corehq.apps.reports.generic import GenericTabularReport
 from custom.ilsgateway.tanzania.reports.utils import link_format
 from dimagi.utils.decorators.memoized import memoized
@@ -17,8 +16,7 @@ from custom.ewsghana.reports.stock_levels_report import StockLevelsReport, Inven
 from custom.ewsghana.reports import MultiReport, EWSData, EWSMultiBarChart, ProductSelectionPane, EWSLineChart
 from casexml.apps.stock.models import StockTransaction
 from django.db.models import Q
-from custom.ewsghana.utils import get_supply_points, make_url, get_second_week, get_country_id, first_item, \
-    ews_date_format
+from custom.ewsghana.utils import get_supply_points, make_url, get_second_week, get_country_id, first_item
 
 
 class ProductAvailabilityData(EWSData):

